textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/constants/LuceneConstants.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * 
+ */
+package edu.uci.ics.textdb.dataflow.constants;
+
+/**
+ * @author sandeepreddy602
+ *
+ */
+public class LuceneConstants {
+    public static final String INDEX_DIR = "index";

```

Comment:
why do we use these value as constants? they are more like the configuration values.